segway/gmtk/input_master.py

```diff
@@ -200,22 +254,70 @@ def get_header_info(self) -> str:
         return " ".join(line)
 
 
-class Mean(OneLineKind):
+class Mean(OneLineArray):
     """
     A single Mean object.
     __init__ and __str__ methods defined in superclass.
     """
     kind = OBJ_KIND_MEAN
 
 
-class Covar(OneLineKind):
+class Covar(OneLineArray):
     """
     A single Covar object.
     __init__ and __str__ methods defined in superclass.
     """
     kind = OBJ_KIND_COVAR
 
 
+class RealMat(OneLineArray):
+    """
+    An entry in a Real matrix object.
+    __init__ and __str__ methods defined in superclass.
+    """
+    kind = OBJ_KIND_RM
+
+    def get_header_info(self) -> str:
+        """
+        Return string representation of own information, for header in
+        input.master section.
+        """
+        # TODO: Should the second 1 be hardcoded? Or is this 2D data?
+        line = [str(len(self)), "1"]  # dimension, additional value
```

Comment:
I'm not entirely sure what the value represents, as the previous implementation hardcoded both items as `1` and there's no documentation on this class. If you don't know, I'll bring it up in our next meeting with Michael. 

My best guess is the data represents a 2-dimensional matrix, which here is a 1 by 1 matrix. If that's the case, I might convert this into a `MultiDimArray` subclass (rather than a `OneLineArray` class) so it can handle matrices larger than 1 by 1. However, I'm not fully sure of this, because the previous implementation was hardcoded as a single line with only a single value.